controller/Delivery.php

```diff
@@ -0,0 +1,432 @@
+<?php
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA ;
+ *
+ */
+
+namespace oat\taoProctoring\controller;
+
+use oat\taoProctoring\controller\Proctoring;
+use oat\taoProctoring\helpers\Breadcrumbs;
+use \core_kernel_classes_Resource;
+
+/**
+ * Proctoring Delivery controllers
+ *
+ * @author Open Assessment Technologies SA
+ * @package taoProctoring
+ * @license GPL-2.0
+ *
+ */
+class Delivery extends Proctoring
+{
+
+    /**
+     * Displays the index page of the deliveries list all available deliveries for the current test center
+     */
+    public function index()
+    {
+
+        $testCenter = $this->getCurrentTestCenter();
+        $deliveries = $this->getDeliveries($testCenter);
+
+        $this->composeView(
+            'deliveries-listing',
+            array(
+                'list' => $deliveries
+            ),
+            array(
+            Breadcrumbs::testCenters(),
+            Breadcrumbs::testCenter($testCenter, $this->getTestCenters()),
+            Breadcrumbs::deliveries($testCenter,
+                array(
+                Breadcrumbs::diagnostics($testCenter),
+                Breadcrumbs::reporting($testCenter)
+            ))
```

Comment:
Not very important, but there is a wrong indentation here, isn't it ? Or maybe it is intentional..